contents/blog/posthog-vs-launchdarkly.md

```diff
@@ -0,0 +1,632 @@
+        <tr>
+            <td><strong>Multivariate flags</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Flags with multiple customizable values</td>
+        </tr>
+        <tr>
+            <td><strong>JSON payloads</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Flags return JSON</td>
+        </tr>
+        <tr>
+            <td><strong>Environments</strong></td>
+            <td className="text-center">Partial</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Manage flags for dev, stage, prod</td>
+        </tr>
+        <tr>
+            <td><strong>Scheduling</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center">Enterprise</td>
+            <td>Schedule flags to turn on or off</td>
+        </tr>
+        <tr>
+            <td><strong>Workflows</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center">Enterprise</td>
+            <td>Manage changes to flags</td>
+        </tr>
+        <tr>
+            <td><strong>Lifecycle management</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Display new and old flags</td>
+        </tr>
+        <tr>
+            <td><strong>Triggers</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Trigger changes based on metrics</td>
+        </tr>
+        <tr>
+            <td><strong>Unlimited targeting size</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center">Enterprise</td>
+            <td>Target contexts of any size</td>
+        </tr>
+        <tr>
+            <td><strong>Early access management</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Manage betas, test features</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+LaunchDarkly’s enterprise plan unlocks advanced workflow features like scheduling, lifecycle management, triggers, and more. PostHog’s [API](/docs/api) enables you to mimic this functionality if needed, but it isn’t built into the UI.
```

Comment:
Even saying via the API was a bit of a stretch. It would require someone to build a mini-app to do scheduling, lifecycle management or triggers. We provide the details they would need (basically flag creation date), but they have to set up all the logic.